crates/base-db/Cargo.toml

```
@@ -1,7 +1,8 @@
 [package]
 name = "base-db"
 version = "0.0.0"
-description = "TBD"
+repository = "https://github.com/rust-lang/rust-analyzer"
```

```suggestion
repository.workspace = true
```
Nit, this should be workspace inehritable no?

---

Now that you mention it, yes of course! Fixed.